Oculo/HealthkitManager.swift

```diff
@@ -129,17 +129,20 @@ class HealthKitManager {
         }
     }
 
-    public func retrieveWalkingStepLength(period : Period, completion : @escaping (Double) -> Void) { // 건강앱의 보폭데이터를 받아오는 메소드
-        guard let walkingStepLengthType = HKSampleType.quantityType(forIdentifier: .walkingStepLength) else { return } //보폭데이터를 받아올것이라는타입 지정
+    /// 건강 앱의 보폭 데이터를 받아오는 메소드
+    public func retrieveWalkingStepLength(period : Period, completion : @escaping (Double) -> Void) {
+
+        /// 보폭 데이터를 받아올 것이라는 타입 지정
+        guard let walkingStepLengthType = HKSampleType.quantityType(forIdentifier: .walkingStepLength) else { return }
         
-        let currentDate = Date()  //현재 시각
-        let startDate = calPeriod(period: period)  // 쿼리문 전송 시작 시각
+        let currentDate = Date()  /// 현재 시각
+        let startDate = calPeriod(period: period)  /// 쿼리문 전송 시작 시각
         let predicate = HKQuery.predicateForSamples(withStart: startDate, end: currentDate, options: .strictStartDate
```

Comment:
변수 이름을 predicate로 설정하신 이유가 있나요? 주석에 시각에 관련된 변수라고 해서 predictDate에서 오타난게 아닌가 싶어서 남겨봅니다~
```suggestion
        let predicate = HKQuery.predictDateForSamples(withStart: startDate, end: currentDate, options: .strictStartDate
```

Reply:
> 변수 이름을 predicate로 설정하신 이유가 있나요? 주석에 시각에 관련된 변수라고 해서 predictDate에서 오타난게 아닌가 싶어서 남겨봅니다~

아하 이 부분은 제가 작성한 코드가 아니라 변수명은 미처 팔로우업 하지 못했습니다.
@mizz0224 에게 물어보고 처리하겠습니다.

Reply:
> 변수 이름을 predicate로 설정하신 이유가 있나요? 주석에 시각에 관련된 변수라고 해서 predictDate에서 오타난게 아닌가 싶어서 남겨봅니다~

원래 메서드가 샘플에 대한 설명(predicate) 부분이라 predicate라는 이름을 쓴 모양입니다!
문제 없어 보입니당!